knowledge_repo/post.py

```diff
@@ -28,6 +28,7 @@
 - example
 created_at: 2016-06-29
 updated_at: 2016-06-30
+users_allowed_to_view: sally_smarts, wesly_wisdom
```

Comment:
What do you think about leaving it out of the template for now since it's an edge case internally, and writing it up in the REAMDE tips & tricks? Thinking we might end up fielding a bunch of confused people on slack otherwise.